portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['I like mint green', 'I have 2 dogs', 'I go to Howard University', 'I live in Maryland'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+
+function randomPepperImage(){
+    const imageIndex1 = Math.floor(Math.random() * 5) + 1;
```

Comment:
It's better to use a global constant variable to control the number of total images so that It's easy to change in future.

Reply:
Also, it's good to have a docstring describing the function.

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Reply:
When I make the variable global, only one image is generated instead of multiple. I think it's because when I press the button, only the function is called and not another number. Any advice?

Reply:
Do you mean same one image is generated every time you click the button? This function doesn't generate multiple images.

Reply:
Yes the button is meant to generate a new image everytime it is clicked, but only generates the same image unless I refresh the page.